language/e2e-testsuite/src/tests/verify_txn.rs

```diff
@@ -301,11 +301,11 @@ fn verify_simple_payment() {
         .script(Script::new(
             p2p_script.clone(),
             vec![account_config::xus_tag()],
-            vec![TransactionArgument::U64(42); MAX_TRANSACTION_SIZE_IN_BYTES as usize],
+            vec![TransactionArgument::U8(42); MAX_TRANSACTION_SIZE_IN_BYTES as usize],
```

Comment:
Not sure if I quite follow this change. Is this to make sure that max transaction size is getting checked before the bytecode verifier is being run?

Reply:
Nothing complicated like that. I should have left a comment in the PR description to explain. This code is just padding out the size of the transaction to trigger the EXCEEDED_MAX_TRANSACTION_SIZE error. If you want to increase the size by MAX_TRANSACTION_SIZE_IN_BYTES, then you should add that many bytes. Adding a vector of that many U64 values adds 8x more space than necessary.